_collections/_section_release_notes2021/11.7.md

```diff
@@ -5,7 +5,17 @@ position: 59
 Release notes
 
 ##### New Functionality
-* We have updated certificate pinning in this release as the current certs will expire in September. You have until then to upgrade to this latest version. Otherwise, bookings will not go through.
+
+Details of what changed for the Certificate Pinning:
+
+Our API will replace an expired certificate with a new one. We use SSL certificate pinning as a security step which means we needed to update the certificate signature on the SDK as well.
+</br>
+* <b>What you need to do</b>
+</br>Just upgrade the SDK to version 11.7.0, no additional step required
```

Comment:
I'd change: Just upgrade the SDK to version 11.7.0, no additional step required -> Update the SDK to version 11.7.0, no additional steps are required.